### ISSUE TITLE
Separation of tile-service into service WAR and rendering lib JAR

### ISSUE THREAD

**Issue by robharper** [oculusinfo/aperture-tiles#118]
The tile-service sub-project currently handles two distinct functions:

1. Handling requests with a servlet, restlet, caching, etc
2. Turning tile data objects into rendered results

The second function could be separated out into a library subproject to allow rendering of tile data outside of a webserver. This would mean one could create a desktop GUI app that shows tiles, a commandline app that writes tile JPG files, or incorporate the renderer library into an existing project with it's own webserver (e.g. based on Spring) that may serve tiles by means other than TMS (sockets, WMS, etc).

**Comment by robharper** [comment]
This is done, will submit a PR once #117 is merged into develop.
